coordinator.go

```diff
@@ -40,6 +41,9 @@ func (t *table) isDeletable() bool {
 		if child.parentOnDeleteAction == deleteActionNoAction && child.deleter.status != statusCompleted {
 			return false
 		}
+		if child.hasGlobalIndex && child.deleter.status != statusCompleted {
```

Comment:
nit: Can we leave some comments to explain why this check is needed? It would be tricky for people who don't know the current limitation of PDML.

Reply:
Sure, I've added a comment here.